doc/Changelog.md

```diff
@@ -3,6 +3,8 @@ Changelog {#Changelog}
 
 # git master
 
+* [45](https://github.com/BlueBrain/Brion/pull/45):
+  Made targets optional
```

Comment:
A bit more elaborate because it's for the user: brain::Circuit::getIDs() does not require targets anymore or something similar